### ISSUE TITLE
About scheduler

### ISSUE THREAD

**Issue by kaneyxx** [monologg/BIO-R-BERT#1]
Hi, there. Thanks for sharing these awesome Bert experiments. I'm reproduce this Bio-R-BERT experiments in DDI corpus by myself. I'm trying to craft without your auto-scripts. Here's a question: Did you setup any hyper-parameters for scheduler?

**Comment by monologg** [comment]
Hi:) I didn't touch any other parameter except seed:) You may try change the hyperparameter for scheduler to get a better result.

**Comment by kaneyxx** [comment]
> Hi:) I didn't touch any other parameters except seed:) You may try changing the `warmup_steps` for scheduler to get a better result.

Thanks for replying so quickly!!!
I'm really new to NLP and PyTorch. I will try my hard to do this. 
Hope I didn't bother you too much :P

**Comment by monologg** [comment]
If you have any issue or question, feel free to ask:)
